superset/views/database/api.py

```diff
@@ -50,3 +137,144 @@ class DatabaseRestApi(DatabaseMixin, BaseSupersetModelRestApi):
+                      foreignKeys:
+                        type: array
+                        description: Table list of foreign keys
+                        items:
+                          type: object
+                          properties:
+                            column_names:
+                              type: array
+                              items:
+                                type: string
+                            name:
+                              type: string
+                            options:
+                              type: object
+                            referred_columns:
+                              type: array
+                              items:
+                                type: string
+                            referred_schema:
+                              type: string
+                            referred_table:
+                              type: string
+                            type:
+                              type: string
+                      indexes:
+                        type: array
+                        description: Table list of indexes
+                        items:
+                          type: object
+                          properties:
+                            column_names:
+                              type: array
+                              items:
+                                type: string
+                            name:
+                              type: string
+                            options:
+                              type: object
+                            referred_columns:
+                              type: array
+                              items:
+                                type: string
+                            referred_schema:
+                              type: string
+                            referred_table:
+                              type: string
+                            type:
+                              type: string
+                      primaryKey:
+                        type: object
+                        properties:
+                          column_names:
+                            type: array
+                            items:
+                              type: string
+                          name:
+                            type: string
+                          type:
+                            type: string
+            400:
+              $ref: '#/components/responses/400'
+            401:
+              $ref: '#/components/responses/401'
+            404:
+              $ref: '#/components/responses/404'
+            422:
+              $ref: '#/components/responses/422'
+            500:
+              $ref: '#/components/responses/500'
+        """
+        table_name_parsed: Optional[str] = parse_js_uri_path_item(table_name)
+        schema_parsed: Optional[str] = parse_js_uri_path_item(
+            schema_name, eval_undefined=True
+        )
```

Comment:
As the return type is already typed in `parse_js_uri_path_item`, do these need to be explicitly typed here?

Reply:
yes, good question. Not really I guess, what do you think?

Reply:
I think it's already implied i.e.not needed, so would personally remove it unless it makes the code more understandable.

Reply:
removed